packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -563,13 +580,15 @@ export const ChatScreen = () => {
         <View ref={chatContainerRef} onLayout={measureChatContainerTop}>
           <KeyboardAvoidingView
             keyboardShowingOffset={
-              hasCurrentlyPlayingTrack ? PLAY_BAR_HEIGHT : 0
+              hasCurrentlyPlayingTrack
+                ? PLAY_BAR_HEIGHT + BOTTOM_BAR_HEIGHT
+                : BOTTOM_BAR_HEIGHT
             }
             style={[
               styles.keyboardAvoiding,
-              hasCurrentlyPlayingTrack
-                ? { bottom: PLAY_BAR_HEIGHT, paddingTop: PLAY_BAR_HEIGHT }
-                : null
+              {
+                bottom: getPlaybarAvoidingBottomPadding()
```

Comment:
we lost the "paddingTop" style, is it ok without it?

Reply:
yep! idk what it was doing

Reply:
It was to fix an alignment issue - setting the bottom of the container doesn't affect the height but it does move it up, so things at the top inside the container would appear underneath the header or nudged up the height of the playbar. Try checking Android on an empty chat

Reply:
damn thanks for the catch!